docs/support/signinissues.md

```diff
@@ -1,83 +1,17 @@
-# Support
+# Problems with Signing In?
 
-## Reset password - Vendor
-If your account is locked or forget your password, you can follow prompts to unblock yourself and get back to your account. 
-Go to https://aka.ms/sspr.
+## Infinite Sign In Loop
+Your account might be disabled if you encounter an issue where you've signed in but the system keeps routing you back to the page.
 
-## Reset password - Public Officer
-If your account is locked or forget your password, you may attempt accessing https://aka.ms/sspr to reset. Please contact your organisation's AFM tech support if there are problems.
+!> Your account can be disabled due to inactivity of 90 days.  
+You should receive email alerts to sign in at least once after 60 days of inactivity.
 
-## Reset MFA - new device
-If you have a new mobile device, you'll need to set it up to work with multi-factor verification. This is a multi-step solution.
+If you are certain that you still need TechPass to access the SG-TechStack integrated services and that you should still have valid access. Please contact the respective TechStack Product team or us [here](support/overview?id=need-more-help) for help.
```

Comment:
Use "SG TechStack" to be consistent with the rest of the documents.

Reply:
done